libexec/rtld-elf/rtld_c18n_policy.txt

```diff
@@ -40,9 +40,13 @@ trust
 	strnstr
 	__libc_start1
 	setjmp
+	longjmp
 	_setjmp
+	_longjmp
 	sigsetjmp
+	siglongjmp
 	unw_getcontext
+	unw_setcontext
```

Comment:
I don't think this function exists, and making the resume call trusted is probably not necessary since we pass it a context?

Reply:
Good catch. Still it should be trusted because otherwise the temporary registers will get cleared when the resume call returns via a trampoline. (Not a problem for C++ exceptions but, as we discussed, it breaks the libunwind API.)

Reply:
Isn't that only the case if we call the function _after_ we've restored the registers? That is to say, if we first call `_rtld_unw_setcontext` and then restore the rest of the registers, clearing whatever was in them before wouldn't matter since we're going to clobber them anyway.

Reply:
I think you are right. Thanks!